features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/data/AuthManager.js

```diff
@@ -178,18 +191,19 @@ class AuthManager {
      * Revoke the issued OAuth access token for currently logged in user and clear both cookie and localstorage data.
      */
     logout() {
-        let authHeader = this.bearer + AuthManager.getUser().getPartialToken();
+        let authHeader = "Bearer " + AuthManager.getUser().getPartialToken();
         //TODO Will have to change the logout end point url to contain the app context(i.e. admin/store, etc.)
-        let url = this.host + "/login/logout/admin";
+        let url = Utils.getAppLogoutURL()
         let headers = {
             'Accept': 'application/json',
             'Content-Type': 'application/json',
             'Authorization': authHeader
         };
         const promisedLogout = axios.post(url, null, {headers: headers});
         return promisedLogout.then(response => {
-            Utils.delete_cookie("WSO2_AM_TOKEN_1");
-            localStorage.removeItem("wso2_user");
+            Utils.delete_cookie(User.CONST.WSO2_AM_TOKEN_1, Utils.CONST.CONTEXT_PATH);
+            localStorage.removeItem(User.CONST.LOCALSTORAGE_USER);
+            new APIClientFactory().getAPIClient(Utils.getEnvironment().label)._instance = null; // Single client should be re initialize after log out
```

Comment:
I think this is not correct, if `._instance` intended to be a private variable in APIClient class, you should not set it in this manner, use a class method to flush the current APIClient instance for given environment.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4890/commits/889b3933af3afb8baa807e737d042ba2c27b27a3